charts/node/values.yaml

```diff
@@ -1,272 +1,333 @@
+extraLabels: {}
 
-## Image of Polkadot Node.
-##
+# -- Image of Polkadot Node.
 image:
+  # -- Image repository
   repository: parity/polkadot
+  # -- Image tag
   tag: latest
+  # -- Image pull policy
   pullPolicy: Always
-  ## Add `-x` shell option to container
-  ## Note: passwords and keys used in container can appear in logs
-  ##
-  debug: false
-
-## Image of Init containers to download chain
-##
-initContainer:
-  image:
-    repository: paritytech/lz4
-    tag: latest
-  ## Add `-x` shell option to initContainers
-  ## Note: passwords and keys used in initContainers can appear in logs
-  ##
+  # -- Adds `-x` shell option to container.
+  # Note: passwords and keys used in container may appear in logs
   debug: false
 
-## Image of kubectl to use for relay P2p service
-##
-kubectl:
-  image:
-    repository: paritytech/kubetools-kubectl
-    tag: latest # Set at 1.22.15 Kubectl version
-
-## Used to sync chain from/to GCS bucket
-##
-googleCloudSdk:
-  image:
-    repository: google/cloud-sdk
-    tag: slim # more lightweight than the full image and still contains gsutil
-  gsutilFlags: '-m -o "GSUtil:parallel_process_count=$(nproc --all)" -o "GSUtil:parallel_thread_count=2" -o "GSUtil:use_gcloud_storage=True"'
-  # serviceAccountKey: ""
-
-## initContainers
-retrieveServiceInfo:
-  resources: {}
-
-syncChainGCS:
-  resources: {}
-
-syncRelayChainGCS:
-  resources: {}
-
-downloadChainspec:
-  resources: {}
-
-downloadRuntime:
-  resources: {}
-
-persistGeneratedNodeKey:
-  resources: {}
-
-injectKeys:
-  resources: {}
-##
-
-## Reference to one or more secrets to be used when pulling images
-## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
-##
+# -- Additional init containers
+initContainers:
+  # -- A container to handle network configuration of the Polkadot node
+  retrieveServiceInfo:
+    image:
+      # -- Image repository
+      repository: paritytech/kubetools-kubectl
+      # -- Image tag
+      tag: latest
+    # -- The resources requests/limits for the container
+    resources: {}
+    # -- Adds `-x` shell option to container.
+    # Note: passwords and keys used in container may appear in logs
+    debug: false
+
+  downloadChainSnapshot:
+    # -- A container to use for downloading a node backup/snapshot
+    image:
+      # -- Image repository
+      repository: docker.io/rclone/rclone
+      # -- Image tag
+      tag: latest
+    # -- The resources requests/limits for the container
+    resources: {}
+    # -- Additional environment variables to add to the container
+    extraEnvVars: []
+    # -- Adds `-x` shell option to container.
+    # Note: passwords and keys used in container may appear in logs
+    debug: false
+    # -- Flags to add to the CLI command. We rely on rclone for downloading snapshots so make sure the flags are compatible.
+    cmdArgs: ""
+
+  downloadChainspec:
+    image:
+      # -- Image repository
+      repository: paritytech/kubetools-kubectl
```

Comment:
For the downloadChainspec and downloadRuntime init container as we are only using `sh` and `wget`, it should be possible to only use `alpine:latest`.